src/org/jitsi/impl/neomedia/MediaStreamImpl.java

```diff
@@ -676,6 +677,15 @@ private void enableRTPExtension(byte extensionID, RTPExtension rtpExtension)
             {
                 absSendTimeEngine.setExtensionID(effectiveId);
             }
+
+            RemoteBitrateEstimatorWrapper remoteBitrateEstimatorWrapper
```

Comment:
we don't actually need to declare this as a `RemoteBitrateEstimatorWrapper` here, right?  we probably only need the `RemoteBitrateEstimator` type?

Reply:
The `RemoteBitrateEstimator` interface doesn't have a `setExtension` method (nor should it).